pkg/resources/table_grant.go

```diff
@@ -42,28 +42,24 @@ var tableGrantSchema = map[string]*schema.Schema{
 		Description:  "The privilege to grant on the current or future table.",
 		Default:      privilegeSelect.String(),
 		ValidateFunc: validation.ValidatePrivilege(validTablePrivileges.ToList(), true),
-		ForceNew:     true,
```

Comment:
I might be missing something but it seems to me we should keep `ForceNew = true` here. The update function doesn't currently know how to handle `privilege` or `on_future` updates.

Reply:
yes, we should keep this as is. reverting now.